### ISSUE TITLE
oci db database create with --wait-for-state not waiting

### ISSUE THREAD

**Issue by amaura** [oracle/oci-cli#400]
Hi, I think a found a bug in the command oci db database create and the --wait-for-state argument. Here is my command :
```
oci db database create \
--wait-for-state AVAILABLE \
--admin-password $PASSWD \
--db-home-id $HOME_ID \
--db-system-id $DBSYS_ID \
--db-name TSTAMF2 \
--db-workload OLTP \
--db-version 19.0.0.0 \
--compartment-id $COMP_ID
```
Problem is that it does not wait for the state AVAILABLE. Command returns immediately with a json containing :
```
...
"lifecycle-state": "PROVISIONING",
...
```

Please note I'm creating this database on an Exadata Cloud. It seems to have the same behaviour on Exadata Cloud at Customer.

**Comment by amaura** [comment]
Also another problem, output is sometimes not the right DB. After the command in my first comment, I terminated the DB and I created a new DB (TSTAMF3). Output was TSTAMF2, with the lifecycle : TERMINATED.

I've also implemented a dirty workaround in go : https://github.com/amaura/oci_database/ and it works, so problem is not on the API backend.

**Comment by kernleee** [comment]
Fixed with 2.24.3, closing.
